tobago-example/tobago-example-demo/src/main/webapp/WEB-INF/web.xml

```diff
@@ -68,16 +68,16 @@
   </listener>
 
   <!-- activate this for some containers -->
-  <!--
```

Comment:
Sorry, this is not very helpful. The demo application should run with myfaces and mojarra without any modification. Because for example I have to test tobago against myfaces and mojarra.

Reply:
Well this was in place before the change, feel free to optimize it.